src/Servers/Kestrel/Transport.Libuv/src/Internal/ListenerContext.cs

```diff
@@ -156,6 +158,13 @@ private UvPipeHandle AcceptPipe()
             return pipe;
         }
 
+        protected void OnThreadShutdown(Exception ex)
+        {
+            ThreadFatalError = ex;
+            StopAcceptingConnections();
```

Comment:
If we move the logging into LibuvThread, this can just be StopAcceptingConnections().

Reply:
Makes sense. I'll make the change.

Reply:
Hi, I made the changes as I understand it. I'm not sure how I would still call StopAcceptingConnections if i'm throwing in startasync. Please let me know if I misunderstood!